tests/module/checks/test_git.py

```diff
@@ -0,0 +1,50 @@
+from pkgcheck.checks import git as git_mod
+from .. import misc
+from pkgcheck.git import _GitCommit
+
+def mk_commit(
+        message,
+        commit="7f9abd7ec2d079b1d0c36fc2f5d626ae0691757e",
+        author="author@domain.com", committer="committer@domain.com",
+        commit_date="Sun Dec 8 02:13:58 2019 -0700",
+    ):
+    return _GitCommit(commit, commit_date, author, committer, message)
+
+
+class TestGitCheck(misc.ReportTestCase):
+    check_kls = git_mod.GitCommitsCheck
+    check = git_mod.GitCommitsCheck(None)
+
+    @staticmethod
+    def commit(message, **kwargs):
+        """Helper function to create commits and cleanup message
+
+        Due to developer convenience, errant leading whitespace is
+        in the messages- this will strip that out so tags are properly
+        parsed."""
```

Comment:
Generally prefer docstring end marks to be on their own line.

Reply:
For multiline docstrings, that is.

Reply:
Ack- works for me.